content/TruenasApps/EnterpriseApps/AsigraApp.md

```diff
@@ -0,0 +1,280 @@
+This basic procedure covers the required Asigra app settings.
+For optional settings, see [Understanding App Installation Wizard Settings](#understanding-app-installation-wizard-settings).
+{{< /hint >}}
+
+{{< include file="static/includes/apps/MultipleAppInstancesAndNaming.md" >}}
+
+{{< include file="/static/includes/apps/LocateAndOpenInstallWizard.md" >}}
+
+{{< trueimage src="/images/SCALE/Apps/InstallAsigraScreen.png" alt="Install Asigra DS-System Screen" id="Install Asigra DS-System Screen" >}}
+
+{{< include file="/static/includes/apps/InstallWizardAppNameAndVersion.md" >}}
+
+Enter the **Asigra Configuration** settings.
+{{< include file="/static/includes/apps/InstallWizardTimezoneSettings.md" >}}
+
+Enter passwords for the database, root user, and the operator user **oper**.
+
+Select the DS-Operator language from the dropdown list. English is the default language.
+
+**Enable Cluster** is selected by default, with five cluster nodes set as the default cluster size.
+Accept the default or change to a value of 3-16 nodes based on your desired configuration.
+
+The TrueNAS app is configured with all the environment variables required to deploy the app.
+If you want to customize the container, click **Add** to the right of **Additional Environment Variables** for each to enter the variable(s) and values(s).
+
+Accept the default values in **Network Configurations**. See [Network Configuration](#network-configuration) for details.
+Leave **Host Network** unselected.
+
+Add your **Storage Configuration** settings.
+
+For **Asigra DS-System Configuration Storage**, set the **Type** to **Host Path (Path that already exists on the system)**.
+Select **Enable ACL**, then enter or browse to and select the **config** dataset to populate the **Host Path** field.
+
+{{< trueimage src="/images/SCALE/Apps/InstallAsigraStorageDataACLandACESettings.png" alt="Add Config Asigra Data ACL and ACE Settings" id="Add Asigra Data ACL and ACE Settings" >}}
+
+Click **Add** to the right of **ACL Entries** for each user or group entry you want to add.
+Click once to add the **0** user ID. Give it **FULL_CONTROL Access**.
+
+Select **Force Flag**.
+
+Next, for **Asigra DS-System Postgres Data Storage**, set **Type** to **Host Path (Path that already exists on the system)**.
+Enter or browse to select the **postgres_data** dataset to populate the **Host Path** field. Do not click **Edit ACL**!
+Select **Automatic Permissions** to set the permissions for this dataset and for the parent dataset if storage volume datasets are nested under a parent dataset. 
+
+{{< trueimage src="/images/SCALE/Apps/InstallAsigraStorageConfigPostgresDataAutoPerms.png" alt="Add Config Asigra Postgres_Data Automatic Permissions" id="Add Asigra Postgres_Data Automatic Permissions" >}}
+
+See [Storage Configuration Settings](#storage-configuration-settings) below for more information.
+
+Accept the defaults in **Resources Configuration**, and select the GPU option if applicable.
```

Comment:
Does Asigra use a GPU? I doubt it.. Should we omit this section entirely?

Reply:
good catch, I'll remove this snippet file and content mentioning GPU. Next commit has the changes.

Reply:
GPU reference should be omitted yes. I'll also remove that section from there, it should be there in the first place